test/exercises/two_fer_test.rb

```diff
@@ -26,6 +26,19 @@ def test_fixtures
   # ###
   # Test the module/class
   # ###
+  def test_string_interpolation_passes
```

Comment:
Could you move this test to around L162 pls? Just to keep the structure of the file.